test/fixtures/prx_story_all.json

```diff
@@ -101,6 +172,10 @@
               "profile": {
                 "href": "http://meta.prx.org/model/audio-file"
               },
+              "prx:storage": {
+                "href": "s3://mediajoint.production.prx.org/public/audio_files/1200648/lcs_spring16_act1.mp3",
+                "type": "audio/mpeg"
```

Comment:
that's a chrome bug, afaict, https://bugs.chromium.org/p/chromium/issues/detail?id=227004
`audio/mpeg` is what I have seen in all the standards docs I checked

If the evap upload has 'audio/mp3', can we fix that in the lambda to be `audio/mpeg`?